src/Microsoft.Azure.SignalR/DependencyInjectionExtensions.cs

```diff
@@ -63,6 +63,7 @@ private static ISignalRServerBuilder AddAzureSignalRCore(this ISignalRServerBuil
                 .AddSingleton(typeof(IServiceConnectionManager<>), typeof(ServiceConnectionManager<>))
                 .AddSingleton(typeof(IServiceEndpointUtility), typeof(ServiceEndpointUtility))
                 .AddSingleton(typeof(ServiceHubDispatcher<>))
+                .AddSingleton(typeof(AzureSignalRMarkerService))
```

Comment:
What will happen if `AddAzureSignalR()` is not called?

Reply:
In the present version, will get exception of 'No service for type xxx has been registered.' 
If add a marker check, will get exception like "Please add all the required services by calling           'IServiceCollection.AddAzureSignalR'"

Reply:
Actually as written it'll null ref since you're calling `GetService` instead of `GetRequiredService`